File: Appfrontend/src/app/models/salaCineDTO.ts
export interface SalaCineDTO {
  id_Sala: number;
  nombre: string;
  estado?: number;
  tipo: string;
  activo: boolean;
}

export interface SalaCineCreateDTO {
  nombre: string;
  estado?: number;
  tipo: string;
}

export interface SalaCineUpdateDTO {
  id_Sala: number;
  nombre: string;
  estado?: number;
  tipo: string;
}

File: API-Cine/API-Cine/Models/DTOs/SalaCineDTO.cs
namespace API_Cine.Models.DTOs
{
    public class SalaCineDTO
    {
        public int Id_Sala { get; set; }
        public string Nombre { get; set; }
        public int? Estado { get; set; }
        public string Tipo { get; set; }
        public bool Activo { get; set; }
    }

    public class SalaCineCreateDTO
    {
        public string Nombre { get; set; }
        public int? Estado { get; set; }
        public string Tipo { get; set; }
    }


    public class SalaCineUpdateDTO
    {
        public int Id_Sala { get; set; }
        public string Nombre { get; set; }
        public int? Estado { get; set; }
        public string Tipo { get; set; }
    }
}
